packages/paraview/env.sh

```diff
@@ -13,23 +13,33 @@ if [ -z "$INSTALLER_PARAVIEW_ENV_SH" ]; then
     name=paraview
     version=5.9.0 
     version_tag=v${version}
+    numpy_version=1.19.2
 
     source_dir_name=paraview-$version_tag
     build_dir_name=build-$version_tag
 
+    declare -A cmake_vars
+
     installer_init
 
-    declare -A cmake_vars
     cmake_vars[BUILD_TESTING]=OFF
     cmake_vars[PARAVIEW_BUILD_SHARED_LIBS]=ON
     cmake_vars[PARAVIEW_BUILD_EDITION]=CATALYST_RENDERING
     cmake_vars[paraview_SOURCE_SELECTION]=$version
     cmake_vars[ENABLE_boost]=ON
+    cmake_vars[USE_SYSTEM_boost]=ON
+    cmake_vars[ENABLE_numpy]=ON
+    cmake_vars[USE_SYSTEM_numpy]=ON
     cmake_vars[ENABLE_protobuf]=ON
```

Comment:
Where is TTK now ? in #5 I update this part to use the one provided by the superbuild